src/dra/d_10798.c

```diff
@@ -17,13 +17,26 @@ s32 D_800B0830[] = {FIX(-2.5), FIX(-2.5), FIX(-2.5)};
 
 s32 D_800B083C[] = {FIX(0), FIX(-0.75), FIX(0.75)};
 
+// tetra spirit out of bounds problem, add a value at beginning and end
+#ifdef VERSION_PC
+u8 D_800B0848[] = {0x00, 0x80, 0x80, 0x80, 0x90, 0x80, 0xA0, 0x80, 0xB0,
+                   0xA0, 0x80, 0xA0, 0x90, 0xA0, 0xA0, 0xA0, 0xB0, 0x03};
+#else
 u8 D_800B0848[] = {0x80, 0x80, 0x80, 0x90, 0x80, 0xA0, 0x80, 0xB0,
                    0xA0, 0x80, 0xA0, 0x90, 0xA0, 0xA0, 0xA0, 0xB0};
+#endif
 
 u16 D_800B0858[] = {0x03C0, 0x0440, 0x0340, 0x04C0};
 
+// tetra spirit out of bounds problem, add another value at end
+#ifdef VERSION_PC
+s16 D_800B0860[] = {31, 0,   31, -32, 0,  -32, -32, -32, -32,
```

Comment:
No need of this here. You can add the `0` at the end on the original array and it will still match

Reply:
Done